gen3/bin/s3.sh

```diff
@@ -6,8 +6,13 @@
 source "${GEN3_HOME}/gen3/lib/utils.sh"
 gen3_load "gen3/gen3setup"
 
+#---------- lib
+
+#
+# Print doc
+#
 gen3_s3_help() {
-  gen3 help s3
+  cat "${GEN3_HOME}/doc/s3.md"
```

Comment:
Keep this `gen3 help s3` - let's let `gen3 help` be the single point of entry to those .md files - treat the .md files like `gen3 help's` internal db ...